scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -250,10 +244,7 @@ task GenerateSitesOnlyVcf {
         String gcs_subnetwork_name
         Boolean leave_cluster_running_at_end
         String hail_version
-        File? hail_wheel
```

Comment:
do we wanna keep the hail wheel option in until we are done tweaking the VAT creation pipeline?

Reply:
There's this ticket to document how to do this (temporarily) for the future, but — according to Kylee and the Hail team — this should not be a permanent part of our pipeline: https://broadworkbench.atlassian.net/browse/VS-1241